security/cmd/node_agent_k8s/main.go

```diff
@@ -334,6 +344,9 @@ func main() {
 	rootCmd.PersistentFlags().DurationVar(&workloadSdsCacheOptions.EvictionDuration, "secretEvictionDuration",
 		24*time.Hour, "Secret eviction time duration")
 
+	rootCmd.PersistentFlags().DurationVar(&serverOptions.RecycleInterval, staledConnectionRecycleIntervalFlag,
```

Comment:
I think this is removed ? (to address comment that ```no need to have both env var and argument at same time```). for other params that have defined both places we may also need to revisit later.

Reply:
Removed. Thanks. Please take a look.